book/layout.md

```diff
@@ -646,10 +652,7 @@ class BlockLayout:
         # ...
 ```
 
-Note that we must add the block's `x` and `y`, since the positions in
```

Comment:
Is this comment no longer relevant?

Reply:
Correct. The block's `x` and `y` are added earlier, when adding the positions to the `self.display_list`.

Reply:
Is that just a "bug" in the text before your PR then?

Reply:
Yes. It seems to have been introduced when we switched `cursor_x` to be a relative measure from the start. (When published it was absolute and switched to relative in Ch7.)